vendors/twtg/models/neon-ts/model.yaml

```diff
@@ -33,7 +33,7 @@ modelId: twtg:neon-ts:1
 # <vendorId>:<protocolName>:<protocolVersion> Needed for linking the model with a specific driver -> must be the same one used in driver.yaml in the corresponding model (You might have several ones)
 protocolId: twtg:ts:1
```

Comment:
protocolId should be updated with the new one

Reply:
Corrected